x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -350,13 +350,12 @@ public void testCanMountSnapshotTakenWhileConcurrentlyIndexing() throws Exceptio
         ensureGreen(restoredIndexName);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/59287")
     public void testMaxRestoreBytesPerSecIsUsed() throws Exception {
         final String repositoryName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         final Settings.Builder repositorySettings = Settings.builder().put("location", randomRepoPath());
         final boolean useRateLimits = randomBoolean();
         if (useRateLimits) {
-            repositorySettings.put("max_restore_bytes_per_sec", new ByteSizeValue(10, ByteSizeUnit.KB));
+            repositorySettings.put("max_restore_bytes_per_sec", new ByteSizeValue(5, ByteSizeUnit.KB));
```

Comment:
Do you think that we could find the "best" value here if we don't randomize anything when indexing documents later in this test so that we have more predictable shard file sizes?

Reply:
Alternatively, could we defer repository creation until after we've created the index and then compute a good value for the throttle based on the index size?

Reply:
Good idea, it also takes care of any future Lucene change that would impact file sizes.